src/Serilog.Sinks.ElmahIO/LoggerConfigurationElmahIOExtensions.cs

```diff
@@ -36,7 +36,7 @@ public static class LoggerConfigurationElmahIOExtensions
         public static LoggerConfiguration ElmahIO(
             this LoggerSinkConfiguration loggerConfiguration,
              Guid logId,
-            LogEventLevel restrictedToMinimumLevel = LogEventLevel.Error,
+            LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
```

Comment:
I've added a home page to the wiki: https://github.com/serilog/serilog-sinks-elmahio/wiki

There's a changelog which I will start filling out when the new version is build.